repository/src/main/java/org/alfresco/service/cmr/search/CategoryService.java

```diff
@@ -234,15 +236,27 @@ public Collection<ChildAssociationRef> getRootCategories(StoreRef storeRef, QNam
      */
     @Auditable(parameters = {"nodeRef"})
     public void deleteCategory(NodeRef nodeRef);
-    
-   /** 
-    * Get the most polular categories 
-    * 
+
+   /**
+    * Get the most polular categories
+    *
     * @param storeRef StoreRef
     * @param aspectName QName
     * @param count int
     * @return List
     */
     @Auditable(parameters = {"storeRef", "aspectName", "count"})
     public List<Pair<NodeRef, Integer>> getTopCategories(StoreRef storeRef, QName aspectName, int count);
+
+    /**
+     * Get a root category NodeRef
+     *
+     * @return NodeRef for category root node
+     */
+    @Experimental
+    @Auditable()

```

Comment:
Missing "storeRef" in Auditable?

Reply:
Yes, thanks - I added that parameter later that the annotation. Fixing this.